frontend/src/APIClients/mutations/TestSessionMutations.ts

```diff
@@ -8,6 +8,14 @@ export const CREATE_TEST_SESSION = gql`
   }
 `;
 
+export const SUBMIT_TEST = gql`
+  mutation CreateTestSessionResult($id: ID!, $result: ResultRequestDTO!) {
```

Comment:
nit: it's not immediately clear what this is an ID for? maybe `testSessionId` or `studentId` or something?